sdk/tests/conformance/textures/misc/origin-clean-conformance-offscreencanvas.html

```diff
@@ -105,13 +109,15 @@
   finishTest();
 }
 
-wtu.setupImageForCrossOriginTest("#img", defaultImgUrl, localImgUrl, imageLoaded);
+(async function() {
+    const img = document.getElementById('img');
+    try {
+        await wtu.awaitOrTimeout(wtu.loadCrossOriginImage(img, defaultImgUrl, localImgUrl), 1000);
+    } catch (e) {
+        console.log(`Image setup failed (${e}), running tests anyway.`);
```

Comment:
The image setup is not supposed to fail given the localhost fallback. Silently running the tests in this case will cause silent flakiness on automated test systems. Could you please change this and the other tests to fail if an exception is thrown at this point?

Reply:
This PR matches the existing behavior, but I'm happy to change the behavior to fail-on-error in a follow-up.

Reply:
OK, sounds fine to leave this as a pure refactoring.